package.json

```diff
@@ -29,6 +29,7 @@
   "author": "Mozilla (https://mozilla.org/)",
   "license": "MPL-2.0",
   "dependencies": {
+    "autoprefixer": "^9.0.1",
```

Comment:
* remove the `^` from all new dependencies
* I think we can remove:

 "grunt-sass": "2.0.0", and ""grunt-autoprefixer": "3.0.4","